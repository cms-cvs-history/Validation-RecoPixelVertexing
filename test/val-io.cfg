process VALIO = {

  source = PoolSource {
    untracked vstring fileNames = {"file:pixel_vertexing.root"}
    untracked int32 maxEvents = 10
    untracked uint32 debugVerbosity = 0
    untracked bool   debugFlag     = true
  }

  module val = PixelVertexVal {
    untracked string OutputFile = "val-io.out"
    untracked uint32 Verbosity = 1
  }

  path p = {val}

}
