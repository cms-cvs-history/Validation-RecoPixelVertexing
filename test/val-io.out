[OVAL] Reconstructed 1 vertexes
[OVAL] z= 0.00155215 +- 0.00277241
[OVAL] Has 10 associated tracks with the following z's: -0.0125661 +- 0.0150201 -0.0108581 +- 0.0090627 -0.00739954 +- 0.007356 -0.00139759 +- 0.00741982 0.00105116 +- 0.007356 0.00230948 +- 0.00726151 0.0029935 +- 0.010362 0.0083962 +- 0.0079799 0.00861913 +- 0.00739029 0.0142502 +- 0.00846296 

[OVAL] Reconstructed 1 vertexes
[OVAL] z= -0.00212778 +- 0.00284377
[OVAL] Has 9 associated tracks with the following z's: -0.0100419 +- 0.007356 -0.00770628 +- 0.0073807 -0.00629576 +- 0.007356 -0.00470095 +- 0.00860444 -0.00130025 +- 0.00745746 0.00321864 +- 0.0072202 0.00447693 +- 0.007356 0.00466809 +- 0.00956119 0.00765803 +- 0.0144899 

[OVAL] Reconstructed 1 vertexes
[OVAL] z= 0.0104909 +- 0.00252504
[OVAL] Has 9 associated tracks with the following z's: -0.00649724 +- 0.007356 0.000727033 +- 0.0072687 0.000798251 +- 0.0073313 0.0116622 +- 0.00963344 0.0129202 +- 0.00721713 0.0130384 +- 0.007356 0.0162589 +- 0.00744559 0.0191285 +- 0.00733831 0.026606 +- 0.00722975 

[OVAL] Reconstructed 1 vertexes
[OVAL] z= -0.0059134 +- 0.00351632
[OVAL] Has 10 associated tracks with the following z's: -0.0278016 +- 0.0169057 -0.0265613 +- 0.00992935 -0.0174269 +- 0.00721712 -0.0157479 +- 0.00834191 -0.00450745 +- 0.00837544 0.000889005 +- 0.0211277 0.00138659 +- 0.0072225 0.00158211 +- 0.007356 0.00498899 +- 0.00721722 0.0324828 +- 0.017503 

[OVAL] Reconstructed 1 vertexes
[OVAL] z= 0.0023314 +- 0.00243756
[OVAL] Has 10 associated tracks with the following z's: -0.0161404 +- 0.00733512 -0.00912514 +- 0.00725921 -0.00874066 +- 0.00724987 -0.0043783 +- 0.00722553 -0.00123949 +- 0.00736895 0.00124156 +- 0.00733447 0.0151122 +- 0.007356 0.0155246 +- 0.00850491 0.0185882 +- 0.00723551 0.0291168 +- 0.010213 

[OVAL] Reconstructed 1 vertexes
[OVAL] z= 0.00912818 +- 0.00308815
[OVAL] Has 9 associated tracks with the following z's: -0.0174252 +- 0.007356 -0.00779537 +- 0.00770917 0.00306762 +- 0.00725932 0.00600227 +- 0.00726413 0.00650052 +- 0.00738701 0.013667 +- 0.00728058 0.020961 +- 0.00735256 0.029954 +- 0.0245497 0.0434861 +- 0.00722156 

[OVAL] Reconstructed 1 vertexes
[OVAL] z= 0.000115339 +- 0.00308987
[OVAL] Has 6 associated tracks with the following z's: -0.0145174 +- 0.00757831 -0.00411188 +- 0.0077898 -0.00320825 +- 0.00728379 0.00117969 +- 0.0073696 0.00433607 +- 0.007356 0.0188019 +- 0.00803416 

[OVAL] Reconstructed 1 vertexes
[OVAL] z= 0.00217986 +- 0.00290545
[OVAL] Has 8 associated tracks with the following z's: -0.0207182 +- 0.00736955 -0.0131024 +- 0.0096765 -0.00231588 +- 0.007356 -0.00184155 +- 0.007356 0.00197567 +- 0.00903345 0.015841 +- 0.00728752 0.0197826 +- 0.00783546 0.021575 +- 0.00982842 

[OVAL] Reconstructed 1 vertexes
[OVAL] z= -0.00139881 +- 0.00275294
[OVAL] Has 8 associated tracks with the following z's: -0.027022 +- 0.00933852 -0.00526065 +- 0.00737899 -0.00282117 +- 0.00773981 -0.00247885 +- 0.00809165 -0.000338217 +- 0.00750398 5.16745e-05 +- 0.0072568 0.0082996 +- 0.00767868 0.00895581 +- 0.00730356 

[OVAL] Reconstructed 1 vertexes
[OVAL] z= -0.010826 +- 0.0038664
[OVAL] Has 7 associated tracks with the following z's: -0.0284762 +- 0.00735193 -0.0174787 +- 0.00918623 -0.00949306 +- 0.0100035 -0.00921906 +- 0.00733946 -0.00396457 +- 0.00734674 -0.000108425 +- 0.00812299 0.0251753 +- 0.022256 

